index.bs

```diff
@@ -3947,15 +3947,22 @@ the attestation=] is consistent with the fields of the attestation certificate's
         public key in the first certificate in |x5c| with the algorithm specified in |alg|.
     - Verify that the public key in the first certificate in |x5c| matches the
         <code>[=credentialPublicKey=]</code> in the <code>[=attestedCredentialData=]</code> in |authenticatorData|.
-    - Verify that in the attestation certificate extension data:
-        - The value of the `attestationChallenge` field is identical to |clientDataHash|.
-        - The `AuthorizationList.allApplications` field is <em>not</em> present, since PublicKeyCredential MUST be [=scoped=] to the
-            [=RP ID=].
-        - The value in the `AuthorizationList.origin` field is equal to `KM_TAG_GENERATED`.
-        - The value in the `AuthorizationList.purpose` field is equal to `KM_PURPOSE_SIGN`.
-    - If successful, return implementation-specific values representing [=attestation type=] [=Basic=] and [=attestation trust
```

Comment:
Looks like this [one of the changes][diff] in #1054 was reverted here. I'll submit a PR to un-revert it.

[diff]: https://github.com/w3c/webauthn/pull/1054/files#diff-ec9cfa5f3f35ec1f84feb2e59686c34dL3870

Reply:
Apolgies, thanks for catching it!

Reply:
No worries! :)